contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html lang = "en">
+	<!-- http://www.w3schools.com/html/html_layout.asp -->
+	
   <title>
   Poodle Security Training School | Contact Us
   </title>
-<body style="font-family:verdana;">
-<!-- http://www.w3schools.com/html/html_layout.asp -->
-<center>
-	<header><h1>Poodle Security Training School</h1></header>
+  <link rel="shortcut icon" href="http://www.hddevelop.com/images/pawprint.ico">
+  <meta charset="utf-8">
+	
+	
+	<style>
+	body{
+	  font-family: Verdana, sans-serif;
+	}
+	
+	.center{
+	  text-align: center;
+	}
+	
+	.bars{
+	  background-color: gray;
+	  color:white;
+	  padding: 25px 0px 25px 0px;
```

Comment:
padding adds extra space around various sides. In this case, the first number (25) refers to the top; second number (0) refers to the right; the third number (25) refers to the bottom; and the fourth number (0) refers to the left.

See here for documentation and interactive tutorials: http://www.w3schools.com/css/css_padding.asp